spec/features/02_user_views_and_edits_profile_page_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+feature "user can view their profile" do
+  scenario "user visits user profile page" do
+    user = FactoryGirl.create(:user)
+    visit '/'
+    click_on 'Sign In'
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
+    click_on 'Log in'
```

Comment:
You should create `sign_in_as` helper method such as [this] (https://github.com/davidrf/adventure_planner/blob/master/spec/support/helpers/authentication_helper.rb). Then you can use this method in your tests. Also, make sure you require the file and include the module in Rspec, take a look at lines 25 and 54 of [this `rails_helper.rb`] (https://github.com/davidrf/adventure_planner/blob/master/spec/rails_helper.rb)